src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFrom-StringData.cs

```diff
@@ -30,19 +30,34 @@ public string StringData
             {
                 return _stringData;
             }
+
             set
             {
                 _stringData = value;
             }
         }
 
+        private string _Delimiter;
```

Comment:
Please remove private field and use auto-implemented property
```c#
 public char Delimiter { get; set; } = '=';

Reply:
Addressed in most recent commit